client/remoterepo.c

```diff
@@ -17,9 +17,109 @@
  *
  * Authors  : Priyesh Padmavilasom (ppadmavilasom@vmware.com)
  */
-
 #include "includes.h"
 
+typedef union {
+    MD5_CTX md5_ctx;
+    SHA_CTX sha1_ctx;
+    SHA256_CTX sha256_ctx;
+} hash_ctx_t;
+
+int
+TDNF_SHA1_Init(
```

Comment:
A single line comment above each method, indicating that these methods are just wrappers would help readers.

Reply:
added comments, those are tdnf wrappers.